### PR TITLE
add writing walkthroughs to getting started

### PR DESCRIPTION
**Pull request by pwright**

### CODE REVIEW

File: docs/content/gs-writing-walkthroughs-proc.adoc [hunk 1]
```diff
@@ -0,0 +1,585 @@
...
+:linkGitHubFork: https://help.github.com/articles/fork-a-repo/
+:linkIndexURL: https://github.com/integr8ly/example-customisations/index.adoc
+:walkthrough-1A: *Integrating message-oriented middleware with a RESTful API using AMQ Online*
+:walkthrough-2: *Managing and protecting APIs using API management*
+:publishing-walkthrough: *Adding walkthroughs to a cluster*
+
+= Writing {PRODUCT} Walkthroughs
+
+
+== About Walkthroughs
```
Comment (pnagesha) [inline docs/content/gs-writing-walkthroughs-proc.adoc]: Is this information only or is this an introduction to all walkthroughs?

File: docs/content/gs-writing-walkthroughs-proc.adoc [hunk 2]
```diff
@@ -0,0 +1,585 @@
...
+
+
+Walkthroughs are tutorials available in {PRODUCT}. The default walkthroughs guide you through the following scenarios:
+
+* {walkthrough-1A}
+* {walkthrough-2}
+
+These walkthroughs are available in the {web-app}.
```
Comment (pnagesha) [inline docs/content/gs-writing-walkthroughs-proc.adoc]: By these you mean the default ones?

File: docs/content/gs-writing-walkthroughs-proc.adoc [hunk 3]
```diff
@@ -0,0 +1,585 @@
...
+
+In addition to the default walkthroughs, you can import existing walkthroughs from a Git repository.
```
Comment (pnagesha) [inline docs/content/gs-writing-walkthroughs-proc.adoc]: I would say something like " To add walkthroughs to your cluster, see <walkthrough name>." The fact that it is is a walkthrough in itself is information that is not needed at this time.

File: docs/content/gs-writing-walkthroughs-proc.adoc [hunk 4]
```diff
@@ -0,0 +1,585 @@
...
+
+* {walkthrough-1A}
+* {walkthrough-2}
+
+These walkthroughs are available in the {web-app}.
+
+In addition to the default walkthroughs, you can import existing walkthroughs from a Git repository. 
+The process for adding walkthroughs to your cluster is itself a walkthrough that is available in the {web-app}.
+
+The following sections describe how to write a walkthrough.
```
Comment (pnagesha) [inline docs/content/gs-writing-walkthroughs-proc.adoc]: This is where I would put the gerund title. "Writing Walkthroughs"

File: docs/content/gs-writing-walkthroughs-proc.adoc [hunk 5]
```diff
@@ -0,0 +1,585 @@
+:linkTroubleshooting: https://github.com/integr8ly/example-customisations/blob/master/docs/troubleshooting.adoc
+:linkGitHubFork: https://help.github.com/articles/fork-a-repo/
+:linkIndexURL: https://github.com/integr8ly/example-customisations/index.adoc
+:walkthrough-1A: *Integrating message-oriented middleware with a RESTful API using AMQ Online*
+:walkthrough-2: *Managing and protecting APIs using API management*
+:publishing-walkthrough: *Adding walkthroughs to a cluster*
+
+= Writing {PRODUCT} Walkthroughs
```
Comment (pnagesha) [inline docs/content/gs-writing-walkthroughs-proc.adoc]: This topic is more than a "how to" I would just call it {Product Walkthroughs}

File: docs/content/gs-writing-walkthroughs-proc.adoc [hunk 6]
```diff
@@ -0,0 +1,585 @@
...
+* {walkthrough-2}
+
+These walkthroughs are available in the {web-app}.
+
+In addition to the default walkthroughs, you can import existing walkthroughs from a Git repository. 
+The process for adding walkthroughs to your cluster is itself a walkthrough that is available in the {web-app}.
+
+The following sections describe how to write a walkthrough.
+
+To write walkthroughs you require a knowledge of:
```
Comment (pnagesha) [inline docs/content/gs-writing-walkthroughs-proc.adoc]: Is there a standard way to call our prerequisites? If I am not mistaken, from the modular doc training, you can use the "Prerequisites" heading.

File: docs/content/gs-writing-walkthroughs-proc.adoc (hunk 1 above)
Comment (pwright) [reply docs/content/gs-writing-walkthroughs-proc.adoc]: good question, no.. its not an introduction to all walkthroughs, that needs to be addressed beforehand. This is about walkthroughs in the context of "now, you want to write your own walkthrough"

File: docs/content/gs-writing-walkthroughs-proc.adoc (hunk 2 above)
Comment (pwright) [reply docs/content/gs-writing-walkthroughs-proc.adoc]: right, I need to add 'default'. We're still debating how to label them in web app, currently we're thinking these are 'Red Hat' walkthroughs, and all others are 'Community' walkthroughs

File: docs/content/gs-writing-walkthroughs-proc.adoc (hunk 3 above)
Comment (pwright) [reply docs/content/gs-writing-walkthroughs-proc.adoc]: hmmm, it kinda is the point here, but I rewrote section to move it into list of walkthroughs

File: docs/content/gs-writing-walkthroughs-proc.adoc (hunk 4 above)
Comment (pwright) [reply docs/content/gs-writing-walkthroughs-proc.adoc]: not sure what you intended, but removed sentence

File: docs/content/gs-writing-walkthroughs-proc.adoc (hunk 5 above)
Comment (pwright) [reply docs/content/gs-writing-walkthroughs-proc.adoc]: The only info that I want to put in here is about 'Writing Walkthroughs', so I'm sure your comment is correct, but I'd prefer to address it by removing or explaining any specious content

File: docs/content/gs-writing-walkthroughs-proc.adoc (hunk 6 above)
Comment (pwright) [reply docs/content/gs-writing-walkthroughs-proc.adoc]: right, I see your point, but it's unreasonable to expect anyone to actually fulfill these as prereqs... that's why I'm going gently here.. maybe 'By the time you have written your first walkthrough, you'll have a knowledge of... ' wdyt?

**Comment by pwright** [comment]
closed in favor of https://github.com/integr8ly/user-documentation/pull/8

**Pull request closed without merging**